staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/util/util.go

```diff
@@ -52,3 +53,8 @@ func ParseEndpoint(endpoint string) (string, error) {
 
 	return u.Path, nil
 }
+
+// ComputeHash returns the SHA256 hash of the given data.
```

Comment:
@enj lmk if there is a better place for this shared function.

Reply:
We can add a small unit test for this. Looking at some of the other packages:

```go
func TestHash(t *testing.T) {
	// hash the empty string to be sure that sha256 is being used
	expect := "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
	sum := ComputeHash("")
	if expect != sum {
		t.Errorf("expected hash %q but got %q", expect, sum)
	}
}
```

Reply:
added

Reply:
> We can add a small unit test for this. Looking at some of the other packages:
> 
> ```go
> func TestHash(t *testing.T) {
> 	// hash the empty string to be sure that sha256 is being used
> 	expect := "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
> 	sum := ComputeHash("")
> 	if expect != sum {
> 		t.Errorf("expected hash %q but got %q", expect, sum)
> 	}
> }
> ```

added

Reply:
It came up in discussing the need for a hash with @enj. There can be keys directly embedded in the config file. So I am thinking of keeping it to sha.

I moved this to the encryptionconfig package where we load config.